src/main/java/com/ericsson/bss/cassandra/ecaudit/auth/ResourceFactory.java

```diff
@@ -53,4 +62,9 @@ public static IResource toResource(String resourceName)
                 throw new IllegalArgumentException("Invalid resource type: " + resourceName);
```

Comment:
Not part of this change, but what happens if this occur? Will it crash the process and therefor it affect traffic, or will it perhaps crash the plugin and therefore is a vector for disabling whitelisting?

Reply:
Ended up creating issue #44 to follow up on this.